test/integration/removeUnwantedIndexes.js

```diff
@@ -0,0 +1,120 @@
+const chai = require('chai');
+const moment = require('moment');
+
+const esClient = require('../../lib/es/esClient');
+const removeUnwantedIndexes = require('../../lib/es/removeUnwantedIndexes');
+const sampleData = require('../resources/sample-data');
+
+const expect = chai.expect;
+const maxWaitTime = 1 * 60 * 1000;
+const orphanAlias = 'profiles-orphan';
+const index1 = `${orphanAlias}_20180117100000`;
+const index2 = `${orphanAlias}_20180117100010`;
+const index3 = `${orphanAlias}_20180117100020`;
+const index4 = `${orphanAlias}_20180117100030`;
+const index5 = `${orphanAlias}_20180117100040`;
+
+const watcherPrefix = '.watcher-history-6-';
+const monitorPrefix = '.monitoring-es-6-';
+
+function getIndexNames(prefix) {
+  const now = moment();
+  return [
+    prefix + now.format('YYYY.MM.DD'),
+    prefix + now.subtract(6, 'days').format('YYYY.MM.DD'),
+    prefix + now.subtract(2, 'days').format('YYYY.MM.DD'),
+  ];
+}
+
+const [todayWatcher, oldestWatcher, expiredWatcher] = getIndexNames(watcherPrefix);
+const [todayMonitor, oldestMonitor, expiredMonitor] = getIndexNames(monitorPrefix);
+
+async function deleteTestIndexes() {
+  await esClient.delete(index1);
```

Comment:
Rather than having a single line for each index deletion could all of the indexes used within the test be added to an array which is iterated to remove all of them? Although both methods are open to forgetting to add an index to be deleted iterating an array feels like it would be less error prone.
Or, is it possible to run a query to get all of the active indexes are delete them all saving any chance of forgetting to delete an index? If this is possible it might be able to be shared between this and the `esClient` integration test.

Reply:
I've added a helper library to delete all indexes.